src/sensors/humidity.rs

```diff
@@ -0,0 +1,37 @@
+//! Module providing humidity sensor functionality.
+
+use super::{LocalisedSensorMetadata, SensorTemplate, SensorTemplateError, Sensors};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
+pub struct HumiditySensor {
+    #[serde(flatten)]
```

Comment:
Oh, that's a nice trick, TIL!

Reply:
We already used it for the `Status` struct. See 28e36109. So TYRL (today you re-learned :wink: )

Reply:
😂